src/_data/glossary.yml

```diff
@@ -81,6 +81,44 @@
     - "language"
     - "type system"
 
+- term: "Bottom type"
+  short_description: |-
+    A type that has no values and is a subtype of all other types.
+  long_description: |-
+    The **bottom type** in Dart is the type that has **no values** and is considered a **subtype of every other type**.
```

Comment:
Consider cross linking the "subtype" text here to [/resources/glossary#subtype](https://dart.dev/resources/glossary#subtype).